src/plugins/dam/components/asset-screen/VariantsTab.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Card, CardBody, CardHeader } from 'reactstrap';
+import damUrl from 'plugins/dam/damUrl';
+import variants from './variants';
+import './variants.scss';
+
+const getVariantSrc = (asset, version) => {
+    const rand = `?r=${(new Date()).getTime()}`;
+    return `${damUrl(asset, version, 'sm')}${rand}`;
+};
+
+export default function VariantsTab({ type, asset }) {
```

Comment:
@meghedi this doesnt do any uploading yet, right?

Reply:
@efjacobson no, not yet.